### PR TITLE
refactor(@ciscospark/i-plugin-support): call getUserToken on submitLogs

### PR DESCRIPTION
**Pull request by saurjai3**
Fixes failure to upload logs from web-client

### CODE REVIEW

File: packages/node_modules/@ciscospark/internal-plugin-support/test/unit/spec/support.js [hunk 1]
```diff
@@ -34,4 +34,11 @@ describe(`plugin-support`, function() {
     });
   });
 
+  describe(`#submitLogs()`, () => {
```
Comment (ianwremmel) [inline packages/node_modules/@ciscospark/internal-plugin-support/test/unit/spec/support.js]: Is there an integration test for this already? Why didn't it catch this?

Comment (saurjai3) [inline packages/node_modules/@ciscospark/internal-plugin-support/test/unit/spec/support.js]: yes there is but it is disabled. The comment says that rackspace is broken. Do you want me to enable that and remove the unit test?
screen shot 2017-06-21 at 4 33 15 pm

Comment (ianwremmel) [inline packages/node_modules/@ciscospark/internal-plugin-support/test/unit/spec/support.js]: ah, gotcha. i remember now. yea, let's try turning that back on.

**Comment by ianwremmel** [comment]
Please turn the disabled test back on, then this is good to merge.

**Pull request merged**